beacon_node/beacon_chain/src/kzg_utils.rs

```diff
@@ -104,7 +108,7 @@ pub fn compute_blob_kzg_proof<E: EthSpec>(
     blob: &Blob<E>,
```

Comment:
There's one on line 99 in `validate_blobs` as well

Reply:
I left that one for 2 reasons:

- `verify_blob_kzg_proof_batch` expects a slice of blobs: `&[Blob]`
- The allocations are being stored in a vec, so they're already on the heap

Reply:
ah got it thanks

Reply:
btw the stack overflow in debug mode suggested that they're on the stack until the `collect` happens (where I'm getting stack overflow).

<img width="834" alt="image" src="https://github.com/user-attachments/assets/dd37afa7-9915-4f53-8e48-d9d09343ae62">

Not related to this PR though and the change is still good!

Reply:
1 blob at a time will definitely be on the stack, but not all of them

Reply:
I guess 1 blob could be enough to stack overflow if there's a bunch of other stuff already on the stack